rotkehlchen/tests/db/test_evmtx.py

```diff
@@ -93,23 +93,28 @@ def test_add_get_evm_transactions(data_dir, username, sql_vm_instructions_cb):
         assert len(errors) == 0
         assert len(warnings) == 0
         filter_query = EvmTransactionsFilterQuery.make(chain_id=ChainID.ETHEREUM)
+    with data.db.conn.read_ctx() as cursor:
         returned_transactions = dbevmtx.get_evm_transactions(cursor, filter_query, True)
         assert returned_transactions == [tx1, tx2]
 
+    with data.db.user_write() as cursor:
```

Comment:
why not opening the cuirsor as

cursor = data.db.conn.cusor() and then close it manually?

Reply:
the context also does the same. And, also commits when they are closed.

Reply:
I mean for the read cursor. That would avoid having to open an close every time. In the test I marked you had many read cursors getting open and close irc

Reply:
yeah for the reader, that should work